app/services/kredits.js

```diff
@@ -105,7 +101,14 @@ export default Service.extend({
       async function instantiateWithAccount (web3Provider, context) {
         console.debug('[kredits] Using user-provided instance, e.g. from Mist browser or Metamask');
         ethProvider = new ethers.providers.Web3Provider(web3Provider);
-        // const network = await ethProvider.getNetwork();
+
+        const network = await ethProvider.getNetwork();
+        if (isPresent(config.web3RequiredNetwork) &&
+            network.name !== config.web3RequiredNetwork) {
```

Comment:
thinking out loud: 
we set `requireSupportedNetwork = false` in environment.js:85 (if WEB3_PROVIDER_URL is set) 
and if it is not false we check if the network is in this list: https://github.com/67P/kredits-contracts/blob/master/lib/kredits.js#L87

Reply:
Yes, happy to review a future PR with that change. This PR is only about requiring the only currently available network, so that the current web app does the right thing.